http-transport-binding.md

```diff
@@ -0,0 +1,338 @@
+into the HTTP request or response body as-is, with the `contentType` attribute
+value declaring its media type; all other event attributes are mapped to HTTP
+headers.
+
+### 1.4. Event Formats
+
+Event formats, used with the *stuctured* content mode, define how an event is
+expressed in a particular data format. All implementations of this
+specification MUST support the [JSON event format][JSON-format], but MAY
+support any additional, including proprietary, formats.
+
+### 1.5. Security
+
+This specification does not introduce any new security features for HTTP, or
+mandate specific existing features to be used. This specification applies
+identically to [HTTP over TLS]([RFC2818][RFC2818]).
+
+## 2. Use of Cloud Events Attributes
+
+This specification does not further define any of the [Cloud Events][CE] event
+attributes.
+
+Two of the event attributes, `contentType` and `data` are handled specially
+and mapped onto HTTP constructs, all other attributes are transferred as
+metadata without further interpretation.
+
+This mapping is intentionally robust against changes, including the addition
+and removal of event attributes, and also accommodates vendor extensions to the
+event metadata. Any mention of event attributes other than `contentType` and
+`data` is exemplary.
+
+### 2.1. contentType Attribute
+
+The `contentType` attribute is assumed to contain a [RFC2046][RFC2046]
+compliant media-type expression.
+
+### 2.2. data Attribute
+
+The `data` attribute is assumed to contain opaque application data that is
+encoded as declared by the `contentType` attribute.
+
+An application is free to hold the information in any in-memory representation
+of its choosing, but as the value is transposed into HTTP as defined in this
+specification, the assumption is that the `data` attribute value is made
+available as a sequence of bytes.
+
+For instance, if the declared `contentType` is
+`application/json;charset=utf-8`, the expectation is that the `data` attribute
+value is made available as [UTF-8][RFC3629] encoded JSON text to HTTP.
+
+## 3. HTTP Message Mapping
+
+The event binding is identical for both HTTP request and response messages.
+
+The content mode is chosen by the sender of the event, which is either the
+requesting or the responding party. Gestures that might allow solicitation of
+events using a particular mode might be defined by an application, but are not
+defined here.
+
+The receiver of the event can distinguish between the two modes by inspecting
+the `Content-Type` header value. If the value is prefixed with the Cloud Events
+media type `application/cloudevents`, indicating the use of a known [event
+format](#14-Event-Formats), the receiver uses *structured* mode, otherwise it
+defaults to *binary* mode.
+
+If a receiver detects the Cloud Events media type, but with an event format that
+it cannot handle, for instance `application/cloudevents+avro`, it MAY still
```

Comment:
I wonder what "handle" means in this case? Do we need to get into the processing model at all?